### PR TITLE
[cinder-csi-plugin] update CSI version

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

update CSI cinder version

note that csi-node-driver-registrar:v2.9.2 is not ready yet
```
# docker pull registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.2
Error response from daemon: manifest for registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.2 not found: manifest unknown: Failed to fetch "v2.9.2"
```

**Which issue this PR fixes(if applicable)**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zetaab** [comment]
perhaps we should wait until all docker images are working from registry?

**Comment by jichenjc** [comment]
ok, will give a try 

2.9.1=>2.9.2 just a CVE update, not some functional , anyway , wait for a few days and let's see :)

**Comment by jichenjc** [comment]
waiting for https://github.com/kubernetes-csi/node-driver-registrar/issues/359

**Comment by jichenjc** [comment]
@zetaab updated, please help take a look, thanks~

**Comment by kayrus** [comment]
@jichenjc can you rebase on latest master? It looks like this PR includes full merge with the latest changes and it's hard to read the actual changes. `git pull --rebase https://github.com/kubernetes/cloud-provider-openstack master` should work without conflicts.

**Comment by jichenjc** [comment]
@kayrus I can see only 4 files changes with 14 lines :) 
I did rebase but https://github.com/kubernetes/cloud-provider-openstack/pull/2478/files seems doesn't have that much change?

**Comment by kayrus** [comment]
@jichenjc hm, probably github somehow cached the diff. Now it looks like it should. Do you need my lgtm?

**Comment by kayrus** [comment]
do we need to update the spec versions here as well?

```sh
$ grep -ri specVersion pkg/
pkg/csi/manila/driver.go:       specVersion   = "1.8.0"
pkg/csi/manila/driver.go:       klog.Info("CSI spec version: ", specVersion)
pkg/csi/cinder/driver.go:       specVersion = "1.8.0"
pkg/csi/cinder/driver.go:       klog.Info("CSI Spec version: ", specVersion)
$ grep -r 1.8.0 docs
docs/cinder-csi-plugin/using-cinder-csi-plugin.md:This plugin is compatible with CSI v1.8.0
```

I'm not sure if this is somehow relevant...

**Comment by jichenjc** [comment]
>>>I'm not sure if this is somehow relevant...

need check more on the spec itself ,e.g whether we really `comply` with the latest spec version 
so I Think it's a different topic and can be a TODO item later @kayrus

**Comment by kayrus** [comment]
@jichenjc let's skip the spec topic for a while, since this variable is only informational.

**Comment by jichenjc** [comment]
@kayrus ok, please help lgtm this if ok to you, thanks

**Comment by jichenjc** [comment]
@zetaab help to approve this ?

**Review by zetaab — APPROVED** [review]
/lgtm

**Comment by jichenjc** [comment]
um... why the merge is not auto-complete? curious

**Pull request merged**
